templates/referent/municipal/volunteer_request.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'referent/_layout.html.twig' %}
+
+{% block referent_content %}
+    <div class="referent__menu-list-event">
+        <div class="menu-items">
+            <h3><a href="{{ path('app_referent_municipal_running_mate_request') }}">Candidats colistiers</a></h3>
+            <span class="separator">|</span>
+            <h3>Candidats volontaires</h3>
+        </div>
+    </div>
+
+    <div id="datagrid"></div>
+{% endblock %}
+
+{% block javascripts %}
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            App.runManagedList(
+                [
+                    {
+                        key: 'lastName',
+                        name: 'Nom',
+                    },
+                    {
+                        key: 'firstName',
+                        name: 'Prénom',
+                    },
+                    {
+                        key: 'phone',
+                        name: 'Tél',
+                    },
+                    {
+                        key: 'favoriteCities',
+                        name: 'Ville(s) choisie(s)',
+                    },
+                    {
+                        key: 'isAdherent',
+                        name: 'Adherent (oui/non)',
```

Comment:
Adhérent

Reply:
Yeah, i'm angry with the accents :sweat_smile: